front_end/src/reducers/TileReducer.js

```diff
@@ -0,0 +1,49 @@
+import {
+  REQUEST_ADD_TILE,
+  RECEIVE_ADD_TILE,
+  REQUEST_TILES,
+  RECEIVE_TILES,
+  REQUEST_TILE,
+  RECEIVE_TILE
+} from 'actions/Tiles/TileActions';
+
+const initialState = {
+  rows: [],
+  details: [],
```

Comment:
Correct, `rows` will hold a list of models, and `details` will be a singular model.